common/helpers_test.go

```diff
@@ -239,7 +240,12 @@ func TestConvertArgument(t *testing.T) {
 	})
 }
 
-func newMockVU(tb testing.TB) *k6modulestest.VU {
+type mockVU struct {
+	*k6modulestest.VU
+	loop *k6eventloop.EventLoop
```

Comment:
@MStoykov Could we expose the event loop on `modulestest.VU`, so that we don't need to wrap it ourselves?

Reply:
You mean as it having another field? I kind of don't want to as it's currently possible to Emulate `RegisterCallback` without an eventloop. If we just add it so that there is a field for it - it seems a bit strange :(. 

modulestest.VU likely needs some redesign as it's currently fairly easy to get a nil exception, so this might end up in it at somepoint ... I guess. But I feel like it currently doesn't really need to.